packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -256,12 +256,15 @@ export class AppMainContainer extends Component<
       isSettingsMenuShown: false,
       unsavedNotebookCount: 0,
       widgets: [],
-      tabs: Object.entries(allDashboardData)
-        .filter(([key]) => key !== DEFAULT_DASHBOARD_ID)
-        .map(([key, value]) => ({
-          key,
-          title: value.title ?? 'Untitled',
-        })),
+      tabs: Object.entries(allDashboardData).map(([key, value]) => ({
+        key,
+        title:
+          key !== DEFAULT_DASHBOARD_ID
+            ? value.title ?? 'Untitled'
+            : 'Code Studio',
```

Comment:
I think you can add this to `dashboardData` in `AppInit` line 168 instead of here. Then it's just `value.title ?? 'Untitled'` for every dashboard